package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appcenter-cli",
-  "version": "1.1.11",
+  "version": "1.1.12",
```

Comment:
You don't need to increate the version (we'll do that when we release so that we can roll up any other changes in a single version number). Please can you revert this change?

Reply:
done!